src/server/project.ts

```diff
@@ -392,8 +392,8 @@ export abstract class Project implements LanguageServiceHost, ModuleResolutionHo
 
     public languageServiceEnabled: boolean;
 
-    readonly trace?: (s: string) => void;
-    readonly realpath?: (path: string) => string;
+    trace?(s: string): void;
+    realpath?(path: string): string;

```

Comment:
Were these changes required to make the flag pass on our codebase, or needed for the PR at all?

Reply:
Actually, doesn't look like this change is necessary anymore. I'll back it out.